.github/pull_request_template.md

```diff
@@ -0,0 +1,72 @@
+# Description
+<!---  Write a brief description of what your code does and how it relates to the issue it is resolving or feature enhancement it is implementing. -->
+
+# Details
+#### URL to JIRA issue
+N/A
+<!---
+  Delete this comment and include the URL of the JIRA issue the pull request is related to.
+  If no JIRA issue exists for this PR, replace this comment with N/A.
+  Your pull request will not pass the required checks if this is not followed.
+-->
+
+#### Link to staging deployment URL 
+<!---
+  Delete this comment and include the URL of the staging environment for this pull request.
+  If no staging environment exists for this PR, replace this comment with N/A and explain why
+  a staging environment is not necessary.
+  Your pull request will not pass the required checks if this is not followed.
+-->
```

Comment:
what if we want to merge in a very simple change (like a change of README file)? Will then our pull request be blocked until we stage the change?

Reply:
It says it there in the second line. Do you think it should be rephrased as it's ambiguous or was it just a mistake of not noticing it?

Reply:
It is a little ambiguous, maybe something like this is better:


```
Delete this comment and include the URL of the staging environment for this pull request, or write "N/A" and explain why a staging environment is not necessary.
Your pull request will not pass the required checks if this is not followed.
```

Reply:
Rephrased. What do you think?